jdaviz/configs/default/plugins/export/export.py

```diff
@@ -130,20 +138,20 @@ def __init__(self, *args, **kwargs):
                                    handler=self._set_subset_not_supported_msg)
         self.session.hub.subscribe(self, SubsetDeleteMessage,
                                    handler=self._set_subset_not_supported_msg)
+        self.session.hub.subscribe(self, DataMessage,
+                                   handler=self._set_dataset_not_supported_msg)

```

Comment:
we should be able to just observe `dataset_selected` and _assume_ that the data remains unchanged once selected (unlike subsets which can change while they're selected).